dashboard/templates/home.html

```diff
@@ -1,5 +1,9 @@
 {% extends 'base.html' %}
 {% load static %}
 {% block content %}
+{% if user.is_superuser %}
+<button style="position:fixed;left:0;top:60px;width:20px;height:20px;background:transparent;border: none !important;"
```

Comment:
As mentioned in the comments I'd probably remove this button and just display this informational div all the time if the user is_superuser.

Reply:
This is just for us for checking build info. I think this will work for now.